src/app/child-dev-project/notes/note-details/note-details.component.ts

```diff
@@ -19,7 +20,14 @@ export class NoteDetailsComponent implements ShowsEntity, OnInit {
   /** interaction types loaded from config file */
   interactionTypes: InteractionType[];
 
-  constructor(private configLoader: NoteConfigLoaderService) {}
+  constructor(
+    private configLoader: NoteConfigLoaderService,
+    private matDialogRef: MatDialogRef<NoteDetailsComponent>
+  ) {
+    this.matDialogRef
+      .beforeClosed()
+      .subscribe(() => this.matDialogRef.close(this.entity));
```

Comment:
What does this do? looks like hooking into the closing process to just manually close it again?